### PR TITLE
Add a sample for the preview tooling support.

### PR DESCRIPTION
**Pull request by zach-klippenstein**
image

### CODE REVIEW

File: compose/.idea/misc.xml [hunk 1]
```diff
@@ -5,6 +5,17 @@
...
+    <option name="filePathToZoomLevelMap">
+      <map>
+        <entry key="../../../../../layout/compose-model-1623893490974.xml" value="0.5503355704697986" />
+        <entry key="../../../../../layout/compose-model-1623951988886.xml" value="0.9939393939393939" />
+        <entry key="../../../../../layout/compose-model-1623952108271.xml" value="0.21388888888888888" />
+        <entry key="../../../../../layout/compose-model-1623952344578.xml" value="0.21388888888888888" />
+        <entry key="../../../../../layout/compose-model-1623952477060.xml" value="0.1" />
+      </map>
+    </option>
+  </component>
```
Comment (zach-klippenstein) [inline compose/.idea/misc.xml:18]: Removed in https://github.com/square/workflow-kotlin/pull/438

**Comment by zach-klippenstein** [comment]
Needs reformatting once https://github.com/square/workflow-kotlin/pull/439 has landed.

**Review by rjrjr — APPROVED** [review]

**Comment by zach-klippenstein** [comment]
Reformatted, now just need a UI test.

- [ ] UI test

File: compose/samples/src/main/java/com/squareup/sample/preview/PreviewActivity.kt [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+  }
+}
+
+@Composable fun PreviewApp() {
+  MaterialTheme {
+    Surface {
+      contactViewFactory.preview(
+        ContactRendering(
+          name = "Dim Tonnelly",
```
Comment (maybeYouShouldBeTheLawyer) [inline compose/samples/src/main/java/com/squareup/sample/preview/PreviewActivity.kt:41]: Always with us...

File: compose/samples/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -35,6 +35,7 @@
     <activity android:name=".hellocomposerendering.HelloComposeRenderingActivity" />
     <activity android:name=".nestedrenderings.NestedRenderingsActivity" />
     <activity android:name=".textinput.TextInputActivity" />
+    <activity android:name=".preview.PreviewActivity" />
```
Comment (maybeYouShouldBeTheLawyer) [inline compose/samples/src/main/AndroidManifest.xml:38]: nit if you want it: looks like these were alphabetized.

**Review by maybeYouShouldBeTheLawyer — APPROVED** [review]
['It's really happening.'](https://www.youtube.com/watch?v=qK5nqmHz0UA&ab_channel=TopClipsTopClips)

File: compose/samples/src/main/AndroidManifest.xml (hunk 3 above)
Comment (zach-klippenstein) [reply compose/samples/src/main/AndroidManifest.xml:38]: - [ ] alphabetize samples

**Comment by zach-klippenstein** [comment]
Fixed the preview cross-hatch in #441.

**Pull request merged**